src/pages/game/Game.tsx

```diff
@@ -1,7 +1,47 @@
-import React from 'react';
+import React, { useCallback, useEffect, useMemo, useRef } from 'react';
+
+import { MainGame } from './utils/game';
+const GAME_HEIGHT = 600;
+const GAME_WIDTH = 800;
 
 const Game = () => {
-	return <>Game page</>;
+  const canvasRef = useRef<HTMLCanvasElement | null>(null);
+  const ctxRef = React.useRef<CanvasRenderingContext2D | null>(null);
+  const game = useMemo(() => new MainGame(GAME_WIDTH, GAME_HEIGHT), []);
+  const lastTimeRef = useRef(0);
+
+  const animate = useCallback(
+    (timestamp = 0) => {
+      const deltaTime = timestamp - lastTimeRef.current;
+      lastTimeRef.current = timestamp;
+      if (ctxRef.current) {
```

Comment:
Не хватает пустых строк между выражениями
```suggestion
      const deltaTime = timestamp - lastTimeRef.current;

      lastTimeRef.current = timestamp;

      if (ctxRef.current) {
```

Стоит наверное отдельно настроить на это линтер

Reply:
Спасибо. Настроил.